src/server/views/index.jade

```diff
@@ -0,0 +1,5 @@
+extends layout
```

Comment:
Why not? If we decide we want to do custom HTML error pages it will come in handy.